doc/api/packages.md

```diff
@@ -1093,9 +1093,11 @@ added: v0.4.0
 }
 ```
 
-The `"main"` field defines the script that is used when the [package directory
-is loaded via `require()`](modules.md#folders-as-modules). Its value
-is a path.
+The `"main"` field defines the CommonJS or ESM script that is used as the
+package entry point.
```

Comment:
I think we should merge the paragraph below that clarifies that `"main"` is not used when there's an `"export"` field: https://github.com/nodejs/node/blob/40ef144b6531e08a26f2690e1f3b6f429ba0cf08/doc/api/packages.md#L1104-L1105
```suggestion
The `"main"` field defines the entry point of a package when
imported by name via a `node_modules` lookup.  Its value
is a path.
When a package has an [`"exports"`][] field, this will take precedence over the
`"main"` field when importing the package by name.
```